### DISCUSSION TITLE
Comments about digital mixers for the community knowledge base

### DISCUSSION THREAD

**Discussion by BentWrench** [jamulussoftware/jamulus#2217] [Documentation]
Hey All,

Wanted to leave a comment about the page "Points to note about inputs" in the community knowledge base here https://jamulus.io/kb/2021/09/09/Inputs.html

I use a Behringer XR18 digital mixer, and it does allow mixing down all 18 channels into 2 channels over USB. I can also route audio from my laptop to the mixer and into Jamulus. The generalization that digital mixers only mix down their signals to their analog outputs does not apply to the XR series of digital mixers from Behringer. Not really a big deal, but wanted to point that out for the other XR users out there that have questions about setting up their mixer for Jamulus.

**Comment by ann0see** [comment]
@pcar75 is the author of this text (which was copied from the wiki). Could you please comment on that/update what needs updating?

**Reply by BentWrench** [reply]
Sure, but I don't see a link or place for comments on that page. Can you give me directions on how to add a comment?

**Reply by ann0see** [reply]
No. Just wait here for his answer. This message was mainly directed to pcar75 ;-).

**Comment by ann0see** [comment]
@BentWrench thank you for this note. It’s not part of the Core part of the site, so we’re not regularly updating this. Probably it’s not possible to give a strong generalisation like that for every mixer. I hope the author of this article reads your post. If you want, I think you can edit the page yourself: https://github.com/jamulussoftware/jamuluswebsite/blob/release/_posts/2021-09-09-Inputs.md

**Comment by pcar75** [comment]
Bonsoir and thanks for the consideration. 

The original comment's intent was to clarify differences between mixers and audio interfaces and was a generalization to which exceptions were stated.  I think it is still a valid note to help people know about audio hardware versus Jamulus , especially entry or low-mid level hardware.  Personally , I woudn't consider the Behringer X18 as an entry mixer.  And of course given time, technology will  fade the differences away.

Note that I am by no means an audio-engineer and you are welcomed to modify the note as you see fit.
Et mon français est bien meilleur que mon anglais (and my french is a lot better than my english) 
:-)
